index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width"/>
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="header">
+        <div class="headerelement headerelement1">Welcome</div>
+        <div class="headerelement headerelement2"><img src="file:///C:/Users/Juan/Documents/GitHub/Portfolio/Images/nav-menu1.png" height="25" width="25" alt="navimg"/></div>
+    </header>
+
+    <div class="headline">
+        <h1><span id="p1">Hey there. I’m Juan</span><br><span id="p2">I’m a software<br>developer</span></h1>
+        <p class="pheadline"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have 
+            a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+    <div class="mediaimgs">
+        <div><img src="file:///C:/Users/Juan/Documents/GitHub/Portfolio/Images/twitter.png" alt="twitter" height="20" width="22"/></div>
+        <div><img src="file:///C:/Users/Juan/Documents/GitHub/Portfolio/Images/linkedin.png" alt="linkedin" height="20" width="22"/></div>
+        <div><img src="file:///C:/Users/Juan/Documents/GitHub/Portfolio/Images/medium.png" alt="medium" height="20" width="22"/></div>
+        <div><img src="file:///C:/Users/Juan/Documents/GitHub/Portfolio/Images/github.png" alt="github" height="20" width="22"/></div>
+        <div><img src="file:///C:/Users/Juan/Documents/GitHub/Portfolio/Images/angellist.png" alt="angellist" height="20" width="22"/></div>
+    </div>
```

Comment:
- [ ] All these should be icons and enclosed using `ul` and `li` tags.

Reply:
Great

Reply:
Well reolved